### ISSUE TITLE
[Feature Request] Option to move suggested spellchecker words back to root of right click context menu.

### ISSUE THREAD

**Issue by lacrimal** [telegramdesktop/tdesktop#7492]
In 1.9.21 version for windows (I'm using windows 10) context spellchecker was moved to 'Spelling' sub menu. This not give optimal performance when you need travel across sub menu to get right spelling suggestion. Also other apps for windows not give us special menu for spelling check since this is main context feature when writing.

Since feel this suboptimal but respect idea of programmers, I will suggest to give users option for moving suggested words to root of right click sub menu, leaving up to user to decide how he want to use it.

More lush solution can be to give more flexibility to manage all right click menu positions in advanced options.

**Comment by lacrimal** [comment]
So, spellchecker moved up in sub menu but this is til the case - still reduced productivity.
Beside that - first two options in spellchecker sub-menu are 'Add to dictionary' and 'Ignore', they are both not common used options so should be placed at the end.

**Comment by rgsiiiya** [comment]
On the Windows desktop app, hiding the spelling list under a sub-menu is a major inconvenience.  I would venture that the #1 task by 99% that is performed when right-clicking on a word is to get to spelling correction words!  These should be the primary presence in the right-click menu when R-clicking a word!  not hidden under a sub-menu.

**Comment by ericrtung** [comment]
I completely agree and am pinging this thread to resurface it. This is incredibly frustrating as its THE ONLY APP that I have ever seen spelling suggestions buried like this.

If nothing else please make it a feature checkbox so users can pick between the buried sub menu, and top of right click menu.
